tests/transaction_test_util.go

```diff
@@ -166,7 +164,8 @@ func verifyTxFields(txTest TransactionTest, decodedTx *types.Transaction) (err e
 		decodedSender common.Address
 	)
 
-	if params.IsHomestead(common.String2Big(txTest.Blocknumber)) {
+	chainConfig := &core.ChainConfig{HomesteadBlock: params.MainNetHomesteadBlock}
```

Comment:
The test used 900000 as the homestead block until now https://github.com/ethereum/go-ethereum/pull/2281/files#diff-293c4ba2fd7888ba7379ee7ec0f69732L94 . Not sure if that's an issue here or not, thought I'd mention it.